docs/examples/players.lua

```diff
@@ -0,0 +1,3 @@
+> Make the player invisible, use only in single player only mods
```

Comment:
I think this (and all the others) are supposed to be --lua comments and not markdown.

Reply:
You can use > and multiple code blocks in .md examples to separate different examples better, or use one .lua example that has to be all lua. But I guess this is just a brainfart.